### ISSUE TITLE
New version of Google Maps SDK for iOS available

### ISSUE THREAD

**Issue by awhitford** [flutter/flutter#32095]
## Use case

When instantiating the `GoogleMap` widget, I see this warning message on my console:
```
Google Maps SDK for iOS version: 2.7.30514.0
New version of Google Maps SDK for iOS available: 3.1.206.0
```

## Proposal

Please update the dependencies.

**Comment by awhitford** [comment]
I just found this article that explains **Upgrading to Google Maps SDK for iOS 3.0**:
* https://medium.com/flutter-io/google-maps-and-flutter-cfb330f9a245

Adding this documentation to this package would be nice.

**Comment by rodriigovieira** [comment]
Do we have any updates on adding this to the documentation?

**Comment by erpankajpatel** [comment]
I have found solutions for this, I have some problems 
 with termial 

->. pod repo remove trunk

-> pod install

-> pod update 

here I have found 
https://stackoverflow.com/questions/58409725/%D0%A1ocoapods-trunk-url-couldnt-be-downloaded

It's worked for me

**Comment by darshankawar** [comment]
I see the message in the console. Verified using latest master and latest plugin version.

```
Launching lib/main.dart on iPhone 12 Pro Max in debug mode...
Running Xcode build...
Xcode build done.                                           12.7s
Debug service listening on ws://127.0.0.1:53572/FoJuYaDZXBk=/ws
Syncing files to device iPhone 12 Pro Max...
Google Maps SDK for iOS version: 3.9.0.0
New version of Google Maps SDK for iOS available: 4.2.0.0
```

**Comment by stuartmorgan** [comment]
Closing as obsolete; the plugin currently doesn't pin a version, which means the newest version compatible with the app's minimum iOS version will be used.
